pages/[channel]/[locale]/category/[slug].tsx

```diff
@@ -25,28 +33,70 @@ export const getStaticProps = async (context: GetStaticPropsContext) => {
       locale: contextToRegionQuery(context).locale,
     },
   });
+
+  const attributesResponse: ApolloQueryResult<FilteringAttributesQuery> = await apolloClient.query<
+    FilteringAttributesQuery,
+    FilteringAttributesQueryVariables
+  >({
+    query: FilteringAttributesQueryDocument,
+    variables: {
+      ...contextToRegionQuery(context),
+      filter: {
+        inCategory: response.data.category?.id,
+      },
+    },
+  });
+
+  let attributes: AttributeFilterFragment[] =
+    attributesResponse.data.attributes?.edges.map((e) => e.node) || [];
+  attributes = attributes.filter((attribute) => attribute.choices?.edges.length);
+
   return {
     props: {
       category: response.data.category,
+      attributeFiltersData: attributes,
     },
   };
 };
 
-function CategoryPage({ category }: InferGetStaticPropsType<typeof getStaticProps>) {
+function CategoryPage({
+  category,
+  attributeFiltersData,
+}: InferGetStaticPropsType<typeof getStaticProps>) {
+  const paths = usePaths();
+  const router = useRouter();
+
   if (!category) {
     return <Custom404 />;
   }
+
+  const subcategories = category.children?.edges.map((edge) => edge.node) || [];
```

Comment:
Why not add a util for extracting those from edges? It's repeated a few times here and there